test_infra/test_files/30001/RUN2/Tomograms/VoxelSpacing3.456/CanonicalTomogram/tomogram_metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "voxel_spacing": 3.456,
-    "reconstruction_method": "",
```

Comment:
Is there any updates required for the v1 db_import related to these metadata updates?

Reply:
Nope, these are just helping the existing test files fit our schema better.